st2debug/st2debug/cmd/submit_debug_info.py

```diff
@@ -211,9 +245,19 @@ def get_system_information():
 
     return system_information
 
+def get_installed_rpm(output_file_path):
```

Comment:
Because this is specific to our needs, would it be possible to specify a set of commands to run in the yaml file instead of directly coding the rpm listing in this module?  Also wondering if the yaml file should be something that we over-write during packaging - because the commands we would want to run would be irrelevant and most likely would fail on any other st2 deployment.